src/Features/CSharp/Portable/ConvertToRecord/ConvertToRecordCommon.cs

```diff
@@ -499,109 +542,130 @@ private static SyntaxList<AttributeListSyntax> GetModifiedAttributeListsForPrope
                             {
-                                if (prevToken.IsKind(SyntaxKind.XmlTextLiteralNewLineToken))
-                                {
-                                    return token.LeadingTrivia;
-                                }
-                                prevToken = token;
+                                return token.LeadingTrivia;
                             }
-                            break;
-                        default:
-                            break;
-                    }
+                            prevToken = token;
+                        }
+                        break;
+                    default:
+                        break;
                 }
-                return null;
             }
+            return null;
         }
 
         private static IEnumerable<XmlNodeSyntax> CreateParamComments(
-            ImmutableArray<PropertyDeclarationSyntax> properties,
+            ImmutableArray<PositionalParameterInfo> propertyResults,
             SyntaxTriviaList exteriorTrivia,
             LineFormattingOptions lineFormattingOptions)
         {
-            foreach (var property in properties)
+            foreach (var result in propertyResults)
             {
-                // get the documentation comment
-                var potentialDocComment = property.GetLeadingTrivia().FirstOrNull(trivia => trivia.IsDocComment());
-                var paramContent = ImmutableArray<XmlNodeSyntax>.Empty;
-                if (potentialDocComment?.GetStructure() is DocumentationCommentTriviaSyntax docComment)
-                {
-                    // get the summary node if there is one
-                    var summaryNode = docComment.Content.FirstOrDefault(node =>
-                        node is XmlElementSyntax element &&
-                        element.StartTag?.Name.LocalName.ValueText == DocumentationCommentXmlNames.SummaryElementName);
+                // add an extra line and space with the exterior trivia, so that our params start on the next line and each
+                // param goes on a new line with the continuation trivia
+                // when adding a new line, the continue flag adds a single line documentation trivia, but we don't necessarily want that
+                yield return SyntaxFactory.XmlText(
+                    SyntaxFactory.XmlTextNewLine(lineFormattingOptions.NewLine, continueXmlDocumentationComment: false),
+                    SyntaxFactory.XmlTextLiteral(" ").WithLeadingTrivia(exteriorTrivia));
 
-                    if (summaryNode != null)
+                if (result.IsInherited)
+                {
+                    // generate a param comment with an inherited doc pointing to the doc comment of the original
+                    // property. If there isn't one it will show up as blank. For prop "Example" from inherited record B:
+                    // <param name="Example"><inheritdoc cref="Base" path="/param[@name='Example']"/></param>
```

Comment:
Do we do this anywhere else or is this the first time?

Reply:
There is a codeFix that will insert an inherit doc comments, but it creates the entire doc comment line and assumes a certain exterior trivia, so it's a bit different. However, from that code and some tests I realized that I don't need the cref or path args, and that the inheritance associates correctly as long as the names of the properties are the same.